### PR TITLE
Add 2x 4x and 0.5x speed selection menu

### PR DESCRIPTION
**Pull request by hvangeffen**
image

### CODE REVIEW

**Comment by hvangeffen** [comment]
Switch icon to mdi-play-speed

**Comment by hvangeffen** [comment]
Add playback speed tooltip

**Comment by hvangeffen** [comment]
Setinterval to settimeout

File: src/components/general/DateTimeSlider.vue [hunk 1]
```diff
@@ -277,6 +302,12 @@ function play(): void {
     dateIndex.value = 0
   }
   increment(playIncrement)
+  if (isPlaying.value) {
```
Comment (hvangeffen) [inline src/components/general/DateTimeSlider.vue:305]: isPlaying can be removed

**Review by vanmeeuwen — APPROVED** [review]

**Pull request merged**